versions.lock

```diff
@@ -70,6 +71,7 @@ org.apache.maven.shared:maven-shared-utils:3.3.4 (1 constraints: e60b61f3)
 org.assertj:assertj-core:3.23.1 (3 constraints: e42af49e)
 org.checkerframework:checker-qual:3.23.0 (3 constraints: 08256088)
 org.checkerframework:dataflow-errorprone:3.23.0 (4 constraints: fa3d685c)
+org.checkerframework:dataflow-nullaway:3.25.0 (2 constraints: 6911b8f1)
```

Comment:
See below, but the mix of 3.23.0 and 3.25.0 versions seems potentially problematic.